src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.c

```diff
@@ -28,61 +46,386 @@ ActorInit En_Takaraya_InitVars = {
     (ActorFunc)EnTakaraya_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ADFB20[] = {
+void* D_80ADFB00[4] = { 0x06006F58, 0x06007358, 0x06007758, 0x06007358 };
+
+void* D_80ADFB10[4] = { 0x06007B58, 0x06007F58, 0x06007758, 0x06007F58 };
```

Comment:
```suggestion
TexturePtr D_80ADFB00[4] = { 0x06006F58, 0x06007358, 0x06007758, 0x06007358 };

TexturePtr  D_80ADFB10[4] = { 0x06007B58, 0x06007F58, 0x06007758, 0x06007F58 };
```
Based on the Draw function, these are texture pointers. That also means these are segmented pointers, so should use the symbols from the object file.